tests/test_aws.py

```diff
@@ -217,6 +280,24 @@ def test_upload_to_container_local_image(self, mock_callback):
         self.assertEqual(self.mock_upload_fileobj.call_count, 0)
         self.assertEqual(obj, result)
 
+    @patch('cloudimg.aws.UploadProgress')
+    def test_upload_to_container_local_image_xz(self, mock_callback):
+        obj = self.mock_object.return_value = MagicMock()
+
+        with NamedTemporaryFile(suffix=".xz") as tmpfile:
```

Comment:
Can we please always provide a meaningful "prefix" when using NamedTemporaryFile?

The motivation is, if you find a lot of files under /tmp being created and not cleaned up, it is useful to be able to rule in/out usage of NamedTemporaryFile as the root cause.

Reply:
Done on both cases.